Objects/object.c

```diff
@@ -2930,6 +2930,39 @@ Py_ReprLeave(PyObject *obj)
 
 /* Trashcan support. */
 
+/* We need to store a pointer in the refcount field of
+ * an object. It is important that we never store 0 (NULL).
+* It is also important to not make the object appear immortal,
+* or it might be untracked by the cycle GC. */
+static uintptr_t
+pointer_to_safe_refcount(void *ptr)
```

Comment:
These two functions are not used when Py_GIL_DISABLED is defined. Is it deliberate?

Reply:
Good point, fixed.

Reply:
Ok, but these functions are still defined even if they are unused, generating warnings:

```
Objects/object.c:2950:1: warning: 'safe_refcount_to_pointer' defined but not used [-Wunused-function]
Objects/object.c:2938:1: warning: 'pointer_to_safe_refcount' defined but not used [-Wunused-function]
```

You need to put these two functions in `#ifndef Py_GIL_DISABLED`.

Reply:
Thanks.